contracts/GnosisSafe.sol

```diff
@@ -116,16 +116,16 @@ contract GnosisSafe
     function execTransaction(
         address to,
         uint256 value,
-        bytes calldata data,
+        bytes memory data,
         Enum.Operation operation,
         uint256 safeTxGas,
         uint256 baseGas,
         uint256 gasPrice,
         address gasToken,
         address payable refundReceiver,
-        bytes calldata signatures
+        bytes memory signatures
     )
-        external
+        public
```

Comment:
Why this change?

Reply:
solidity internals ... different generated stack ... slighly different gas usage